docs/sphinx/source/whatsnew/v0.7.1.rst

```diff
@@ -26,6 +26,8 @@ Testing
 ~~~~~~~
 * Added single-year PSM3 API test for `iotools.get_psm3`.
 * Added tests for `iotools.parse_psm3` and `iotools.read_psm3`.
+* Change test folder to `tests` and reorganize tests into subfolders, _eg_
```

Comment:
```suggestion
* Change `pvlib/test` folder to `pvlib/tests` and reorganize tests into subfolders, _eg_
```

Reply:
PS I had to check the "allow edits from maintainers" box before the "Commit suggestion" button was accessible